pkg/harvester/store/harvester-store/index.ts

```diff
@@ -1,13 +1,16 @@
 //@ts-nocheck
 import { CoreStoreSpecifics, CoreStoreConfig } from '@shell/core/types';
 
+import SteveSchema from '@shell/models/steve-schema';
 import { SteveFactory, steveStoreInit } from '@shell/plugins/steve/index';
 import { PRODUCT_NAME } from '../../config/harvester';
 import getters from './getters';
 import mutations from './mutations';
 import actions from './actions';
 
-const harvesterFactory = (): CoreStoreSpecifics => {
+const harvesterFactory = (config): CoreStoreSpecifics => {
+  SteveSchema.reset(config.namespace);
```

Comment:
this shouldn't be needed. this store inherits the steve mutation `reset` which does this